tests/python/tddft/stab.py

```diff
@@ -0,0 +1,58 @@
+import psi4
```

Comment:
CTest (unless you've added something) doesn't know to look for tests this deep in the directory structure. Maybe move to `tests/python/tddft-stab/input.py`

Reply:
This does not run, so it is okay that CTest won't find it.

I could be removed, however it would be useful code for whoever works on stability analysis down the road, (it makes an instability for you to find/follow)

Reply:
no call to remove it from me. I think it's good to have psithon tests/examples as well as the testing-focused pytest unittests. The former provide a more readable introduction and also are more likely to have multiple jobs in a row, which helps to catch interference btwn jobs.